.github/workflows/ci.yaml

```diff
@@ -214,7 +215,10 @@ jobs:
           platforms: linux/amd64
           tags: ${{ format('{0}/{1}:{2}', steps.login-ecr.outputs.registry, steps.ref.outputs.repo-name, steps.ref.outputs.image-tag) }}
           push: false
+          load: true
           build-args: READONLY_API_TOKEN_GITHUB=${{ secrets.READONLY_API_TOKEN_GITHUB }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
```

Comment:
how did you work out that we need to have these parameters?

Reply:
I learned that the docker/build-push-action has internal caching capabilities and we're not using it. It has an option for github actions, and I'm following the examples here: https://github.com/docker/build-push-action/blob/master/docs/advanced/cache.md#github-cache